test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -57,11 +96,18 @@ export default class PossibleIssues extends Component {
         if (error) {
             return <div>Error: {error}</div>;
         } else {
-            const columns = [{
-                title: 'Possible Issues',
-                dataIndex: 'issue',
-                key: 'issue',
-            }];
+            const columns = [
+                {
+                    title: 'Possible Issues',
+                    dataIndex: 'issue',
+                    key: 'issue',
+                },
+                {
+                    title: 'Related Degree',
+                    dataIndex: 'degree',
+                    key: 'degree',
```

Comment:
Currently we haven't completed the pipeline to automatically collect issues on GitHub repos, so the ML model now is limited to predict only several manually collected issues, instead of covering all possible issues to rank them. For example, in the case of previously posted image, ML only has the info of the first issue, but not the second and third. After completing monitoring all issues, we will be able to rank them.